src/schemas/general/Timeout.ts

```diff
@@ -0,0 +1,14 @@
+import { z } from 'zod';
+
+import { formatDuration } from 'utils';
+import { ONE_MINUTE_IN_MS, ONE_SECOND_IN_MS } from 'utils.constants';
+
+export const MIN_BASE_TIMEOUT = ONE_SECOND_IN_MS;
+export const MAX_BASE_TIMEOUT = ONE_MINUTE_IN_MS;
+
+export const timeoutSchema = (minTimeout = MIN_BASE_TIMEOUT, maxTimeout = MAX_BASE_TIMEOUT) => {
```

Comment:
This is not a schema. It is a schema creator.
IMO the name should be something like `createTimeoutSchema`